package/test_requirements.txt

```
@@ -1,21 +1,21 @@
 -r requirements.txt
 kedro[pandas.ParquetDataSet]>=0.17.0  
 kedro-telemetry>=0.1.1  # for testing telemetry integration
-bandit~=1.6.2
-behave>=1.2.6, <2.0
-black~=22.0
-flake8~=3.9.2
-isort~=5.8.0
-matplotlib~=3.3.4
-mypy~=0.930
-psutil==5.8.0  # same as Kedro for now
-pylint~=2.8.2
-pytest~=6.2.0
-pytest-asyncio~=0.17.2
-pytest-cov~=2.11.1
-pytest-mock~=3.6.1
+bandit~=1.7
+behave>=1.2, <2.0
```

```suggestion
behave~=1.2
```
Just to be consistent with how we write it elsewhere.